packages/core/src/api/exporters/html/__snapshots__/fontSize/basic/external.html

```diff
@@ -1 +1 @@
-<p class="bn-inline-content"><span style="font-size: 18px" data-style-type="fontSize" data-value="18px" data-editable="">This is text with a custom fontSize</span></p>
```

Comment:
class name removed intentionally?

Reply:
Yep I figured it really doesn't add anything to include `bn-` class names since they're only used for block styling and blocks are removed here.